### PR TITLE
🛠️ [TOOLS] Update all message files

### PR DESCRIPTION
**Pull request by juggler31**
## Checklist

- [x] I have followed the guidelines in our [Contributing document](https://geotrek.readthedocs.io/en/latest/CONTRIBUTING.html)
- [ ] My code respects the Definition of done available in the [Development section of the documentation](https://geotrek.readthedocs.io/en/latest/contribute/development.html#definition-of-done-for-new-model-fields)
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes
- [x] I added an entry in the changelog file
- [x] My commits are all using prefix convention (emoji + tag name) and references associated issues
- [x] I added a label to the PR corresponding to the perimeter of my contribution
- [ ] The title of my PR mentionned the issue associated

### CODE REVIEW

File: geotrek/api/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -638,3 +629,14 @@ msgid ""
 msgstr ""
 "Ce champ est déprécié et sera retiré dans de prochaines releases, merci "
 "d'utiliser 'capacity' à la place."
+
+#, fuzzy
```
Comment (submarcos) [inline geotrek/api/locale/fr/LC_MESSAGES/django.po:633]: fuzzy significate that django has tried to translate itmself. Please check and correct translate, then delete fuzzy reference

File: geotrek/trekking/tests/data/file_bad_encoding_tmp.kml
Comment (marcantoinedupre) [inline geotrek/trekking/tests/data/file_bad_encoding_tmp.kml:1]: Why has this file changed?

File: geotrek/trekking/tests/data/file_good_encoding_tmp.kml
Comment (marcantoinedupre) [inline geotrek/trekking/tests/data/file_good_encoding_tmp.kml:1]: Same here: why has this file changed?

File: Makefile
Comment (marcantoinedupre) [inline Makefile:1]: Unrelated? Or maybe in another commit at least?

**Review by marcantoinedupre — CHANGES_REQUESTED** [review]
Nice work! I have spotted a few changes which looks unrelated.

**Review by marcantoinedupre — APPROVED** [review]

File: docs/changelog.rst [hunk 2]
```diff
@@ -5,6 +5,10 @@ CHANGELOG
 2.101.5+dev (XXXX-XX-XX)
 ------------------------
 
+**Minor changes**
+
+- Update all message files
```
Comment (submarcos) [inline docs/changelog.rst:10]:
```suggestion
- Update all translation files
```

File: geotrek/api/locale/fr/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -625,16 +617,10 @@ msgstr ""
...
-msgstr ""
-"Ce champ est déprécié et sera retiré dans de prochaines releases, merci "
-"d'utiliser 'start_time' à la place."
-
-msgid ""
-"This field is deprecated and will be removed in next releases. Please start "
-"using 'capacity'"
+"using '%(field)s'"
 msgstr ""
 "Ce champ est déprécié et sera retiré dans de prochaines releases, merci "
```
Comment (submarcos) [inline geotrek/api/locale/fr/LC_MESSAGES/django.po:625]:
```suggestion
"Ce champ est déprécié et sera retiré dans une future version, merci "
```

File: geotrek/tourism/locale/fr/LC_MESSAGES/django.po [hunk 4]
```diff
@@ -259,6 +259,12 @@ msgstr "contenu-touristique"
 msgid "Published touristic contents"
 msgstr "Contenus touristiques publiés"
 
+msgid "Signages"
+msgstr ""
```
Comment (submarcos) [inline geotrek/tourism/locale/fr/LC_MESSAGES/django.po:263]:
```suggestion
msgstr "Signalétiques"
```

File: geotrek/signage/locale/fr/LC_MESSAGES/django.po [hunk 5]
```diff
@@ -162,9 +180,18 @@ msgstr "Ajouter une ligne"
 msgid "Remove"
 msgstr "Supprimer"
 
+msgid "No result"
+msgstr ""
```
Comment (submarcos) [inline geotrek/signage/locale/fr/LC_MESSAGES/django.po:184]: translation is missing

File: geotrek/signage/locale/fr/LC_MESSAGES/django.po [hunk 6]
```diff
@@ -162,9 +180,18 @@ msgstr "Ajouter une ligne"
...
+
+msgid "Choose value(s)"
+msgstr ""
```
Comment (submarcos) [inline geotrek/signage/locale/fr/LC_MESSAGES/django.po:187]: translation is missing

File: geotrek/signage/locale/fr/LC_MESSAGES/django.po [hunk 7]
```diff
@@ -59,9 +65,21 @@ msgstr "Type de signalétique"
...
+
 msgid "Code"
 msgstr "Code"
 
+msgid "Complete description"
+msgstr "Description complète"
+
+msgid "Line pictogram"
+msgstr "Pictogramme des lignes"
```
Comment (submarcos) [inline geotrek/signage/locale/fr/LC_MESSAGES/django.po:78]:
```suggestion
msgstr "Pictogramme de ligne"
```

File: geotrek/signage/locale/fr/LC_MESSAGES/django.po [hunk 8]
```diff
@@ -59,9 +65,21 @@ msgstr "Type de signalétique"
...
+
+msgid "Line pictograms"
+msgstr "Pictogrammes des lignes"
```
Comment (submarcos) [inline geotrek/signage/locale/fr/LC_MESSAGES/django.po:81]:
```suggestion
msgstr "Pictogrammes de ligne"
```

File: geotrek/signage/locale/fr/LC_MESSAGES/django.po [hunk 9]
```diff
@@ -36,6 +36,12 @@ msgstr ""
 msgid "Provider"
 msgstr "Fournisseur"
 
+msgid "Filter by one or more structure."
+msgstr "Filtrer par un ou plusieurs gestionnaires."
```
Comment (submarcos) [inline geotrek/signage/locale/fr/LC_MESSAGES/django.po:40]:
```suggestion
msgstr "Filtrer par un ou plusieurs gestionnaires."
```

bad translation

File: geotrek/maintenance/locale/fr/LC_MESSAGES/django.po [hunk 10]
```diff
@@ -124,18 +124,17 @@ msgstr "La date de début est après la date de fin"
 msgid "Fundings"
 msgstr "Financements"
 
+#, fuzzy
+#| msgid "Begin date is after end date"
```
Comment (submarcos) [inline geotrek/maintenance/locale/fr/LC_MESSAGES/django.po:128]: fuzzy !

File: geotrek/feedback/locale/fr/LC_MESSAGES/django.po [hunk 11]
```diff
@@ -115,9 +115,18 @@ msgstr "Créer un signalement"
 msgid "Feedback from {email}"
 msgstr "Signalement de {email}"
 
+msgid "to"
+msgstr ""
```
Comment (submarcos) [inline geotrek/feedback/locale/fr/LC_MESSAGES/django.po:119]:
```suggestion
msgstr ""
```

missing translation

File: geotrek/common/locale/fr/LC_MESSAGES/django.po [hunk 12]
```diff
@@ -681,8 +689,7 @@ msgstr "ces champs doivent être renseignés :"
 msgid "has invalid geometry;"
 msgstr "a une géométrie invalide ;"
 
-#, fuzzy, python-format
-#| msgid "%(obj)s has changed  state, it needs a review to be published. "
+#, python-format
 msgid "%(obj)s has changed state, it needs a review to be published. "
 msgstr "%(obj)s a changé d'état, et a besoin d'être review pour être publié."
```
Comment (submarcos) [inline geotrek/common/locale/fr/LC_MESSAGES/django.po:694]:
```suggestion
msgstr "%(obj)s a changé d'état, et a besoin d'être validé pour être publié."
```

File: geotrek/common/locale/fr/LC_MESSAGES/django.po [hunk 13]
```diff
@@ -484,12 +488,16 @@ msgid ""
...
+msgstr ""
+
 #, python-brace-format
 msgid "Model {model}"
 msgstr ""
 
-#, fuzzy
-#| msgid "Geotrek Rando"
 msgid "Geotrek-admin"
 msgstr "Geotrek Admin"
```
Comment (submarcos) [inline geotrek/common/locale/fr/LC_MESSAGES/django.po:502]:
```suggestion
msgstr ""
```

File: geotrek/common/locale/fr/LC_MESSAGES/django.po [hunk 14]
```diff
@@ -484,12 +488,16 @@ msgid ""
 "not tall enough"
 msgstr ""
 
+#, python-brace-format
+msgid ""
+"Invalid attachment file {url} for {self.obj.__class__.__name__} #{self.obj."
+"pk}: "
+msgstr ""
+
```
Comment (submarcos) [inline geotrek/common/locale/fr/LC_MESSAGES/django.po:496]: missing translation

**Pull request merged**